### PR TITLE
INRETS and bpr2 VD functions

### PR DESCRIPTION
**Pull request by Art-Ev**
Just adding INRETS and a BPR2 VD function.
Not fully tested but I think it should work almost as it is.

### CODE REVIEW

**Comment by Art-Ev** [comment]
First styling revision, do we need to correct all E501 error (line too long) ?
Anything to check for pyx and rst files ?

File: aequilibrae/paths/bpr2.pyx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                      double [:] fftime,
+                      double[:] alpha,
+                      double [:] beta,
+                      int cores):
+  cdef long long i
+  cdef long long l = congested_time.shape[0]
+
+  for i in prange(l, nogil=True, num_threads=cores):
+      if link_flows[i] > 0:
+        if link_flows[i] > capacity[i]:
```
Comment (pedrocamargo) [inline aequilibrae/paths/bpr2.pyx:43]: This indentation here looks funny. Can you change it for the PEP8 standard of 4 spaces?

**Comment by pedrocamargo** [comment]
Some documentation on INRETS, @janzill http://transportproblems.polsl.pl/pl/Archiwum/2017/zeszyt1/2017t12z1_04.pdf

File: aequilibrae/paths/inrets.pyx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                      double [:] fftime,
+                      double[:] alpha,
+                      double [:] beta,
+                      int cores):
+  cdef long long i
+  cdef long long l = congested_time.shape[0]
+
+  for i in prange(l, nogil=True, num_threads=cores):
+      if link_flows[i] > 0:
+        if link_flows[i] > capacity[i]:
```
Comment (pedrocamargo) [inline aequilibrae/paths/inrets.pyx:43]: Same here

**Review by pedrocamargo — CHANGES_REQUESTED** [review]
@Art-Ev , it looks excellent, aside from the issue with indentation and the lack of testing.  You can look at this file for reference: https://github.com/AequilibraE/aequilibrae/blob/master/tests/aequilibrae/paths/test_conical.py

**Comment by pedrocamargo** [comment]
I second @janzill 's comment on tests. We are actually expanding the guidelines on collaboration to make it easier, @Art-Ev !

**Comment by Art-Ev** [comment]
Can somebody check my inrets values ? Calculated them with excel but cannot retrieve exactly the same values for Spiess' conical through excel than the values in test_conical (ok at 2 decimals but values differs beyond that)

**Comment by pedrocamargo** [comment]
@Art-Ev , we can go ahead and merge if you change the target from Master to Develop

**Comment by Art-Ev** [comment]
> @Art-Ev , we can go ahead and merge if you change the target from Master to Develop
I thought I did it 14 days ago after Janzill message... github tells me "Art-Ev wants to merge 4 commits into AequilibraE:develop from Art-Ev:INRETS-and-BPR2-DV-functions". Where can I correct this?

**Comment by janzill** [comment]
> > @Art-Ev , we can go ahead and merge if you change the target from Master to Develop
> 
> I thought I did it 14 days ago after Janzill message... github tells me "Art-Ev wants to merge 4 commits into AequilibraE:develop from Art-Ev:INRETS-and-BPR2-DV-functions". Where can I correct this?

I can see the change @pedrocamargo, will need to look into @Art-Ev's question regarding inrets comparison values

**Comment by pedrocamargo** [comment]
@Art-Ev , something went haywire with those indentations.

image

**Review by pedrocamargo — APPROVED** [review]

**Pull request merged**